linera-service/src/cli_wrappers/gcp_net.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use super::{
+    util::ensure_connected_cluster_is_running, ClientWrapper, LineraNet, LineraNetConfig, Network,
+};
+use anyhow::Result;
+use async_trait::async_trait;
+use linera_base::data_types::Amount;
+use std::sync::Arc;
+use tempfile::{tempdir, TempDir};
+
+#[cfg(any(test, feature = "test"))]
+pub struct GCPNetTestingConfig;
+
+#[cfg(any(test, feature = "test"))]
+impl GCPNetTestingConfig {
+    pub fn new() -> Self {
+        Self {}
+    }
+}
+
+#[cfg(any(test, feature = "test"))]
+#[async_trait]
+impl LineraNetConfig for GCPNetTestingConfig {
+    type Net = GCPNet;
+
+    async fn instantiate(self) -> Result<(Self::Net, ClientWrapper)> {
+        let seed = 37;
+        let mut net = GCPNet::new(Some(seed)).expect("Creating GCPNet should not fail");
+
+        let client = net.make_client().await;
+        client.wallet_init(&[], self.maybe_faucet()).await.unwrap();
+
+        for _ in 0..2 {
```

Comment:
What is the repetition for here?

Reply:
This is what I had to do on the shared local kubernetes also. `create-genesis-config` generates the initial chains with some attributes (some initial balance, etc). Since the tests were written assuming this returned client would be the one from when we create the network, they assume these properties of the chains for the tests to pass. I'm basically replicating that here for the tests to pass.

Since I had to do that for shared local kubernetes, I decided to do that also here just in case (once I get it to pass like this, I double check it's needed and remove it in case it isn't, but I didn't get to that part yet 😅).